src/interactive/Toast.jsx

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Icon from '../media/Icon';
+
+export const TOAST_CLASS = 'toast';
+export const TOAST_ACTION_CLASS = `${TOAST_CLASS}-action`;
+export const TOAST_DISMISS_BTN_CLASS = `${TOAST_CLASS}-dismissBtn`;
+export const SUCCESS_TOAST_CLASS = `${TOAST_CLASS}--success`;
+export const ERROR_TOAST_CLASS = `${TOAST_CLASS}--error`;
+export const HIDDEN_TOAST_CLASS = `${TOAST_CLASS}--hide`;
+
+/**
+ * @module Toast
+ */
+class Toast extends React.PureComponent {
+	constructor(props) {
+		super(props);
+
+		this.handleDismiss = this.handleDismiss.bind(this);
+	}
+
+	handleDismiss() {
+		if (this.props.setDismissedToast) {
```

Comment:
you could shorthand this as :)
`this.props.setDismissedToast && this.props.setDismissedToast(this);`